lib/components/Charts/AreaChart/index.tsx

```diff
@@ -25,7 +18,22 @@ export type AreaChartProps<
   DataConfig extends ChartConfig = ChartConfig,
   Keys extends string = InferChartKeys<DataConfig>,
 > = ChartPropsBase<DataConfig, Keys> & {
-  lineType?: "natural" | "linear" | "step"
+  lineType?:
```

Comment:
You might create a type in order no to have a long piece of code with the allowed line types

`type lineType =  "basis"  | "basisClosed" | "basisOpen" | "bumpX" | "bumpY" | "bump"....`

```
  DataConfig extends ChartConfig = ChartConfig,
  Keys extends string = InferChartKeys<DataConfig>,
> = ChartPropsBase<DataConfig, Keys> & {
  lineType?: lineType
```